src/sentry/api/endpoints/monitor_checkins.py

```diff
@@ -23,7 +25,14 @@
 from sentry.apidocs.parameters import GLOBAL_PARAMS, MONITOR_PARAMS
 from sentry.apidocs.utils import inline_sentry_response_serializer
 from sentry.models import CheckInStatus, Monitor, MonitorCheckIn, MonitorStatus, Project, ProjectKey
+from sentry.ratelimits.sliding_windows import Quota, RedisSlidingWindowRateLimiter, RequestedQuota
 from sentry.signals import first_cron_checkin_received, first_cron_monitor_created
+from sentry.utils import metrics
+
+checkin_ratelimiter = RedisSlidingWindowRateLimiter(
+    cluster=settings.SENTRY_RATE_LIMIT_REDIS_CLUSTER
+)
```

Comment:
@beezz This should be the `ratelimiter` cluster in prod, I assume this is fine to use here?

Reply:
Nm, using ratelimiting abstraction instead